### ISSUE TITLE
Roadmap and Co-Maintainership

### ISSUE THREAD

**Issue by knatsakis** [carnager/rofi-pass#236]
Hi @carnager 👋🏻 ,

There seem to be a lot of open issues and PRs that haven't been attended to for a long a time. Is this project still maintained?

Would you be interested in adding a co-maintainer?

Regards!

**Comment by carnager** [comment]
Yes, I agree, this needs some manpower that looks through PRs and Issues. So yeah, I am open for this

**Comment by Seme4eg** [comment]
Hi! Maybe you can mention my [fork](https://github.com/Seme4eg/rofi-pass-wayland) somewhere at the end of your readme? I didn't make a PR because preserving backwards-compat would make the codebase much messier. I've been daily driving my fork for almost half a year now and works good. The 'mess of a config file' you mentioned is kinda resolved in my fork tho i dropped X support, which might be turning point for many.
